.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### Current Behavior
+
+- What code are you running and what is happening?
+- Include a screenshot if it makes sense.
```

Comment:
I'd not make "adding a screenshot" a separate item since it's not necessary in >90% of reports.